package task1;
option optimize_for = SPEED;

// Event of Poisson Process. Server -> Client.
message Event {
    optional int64 server_timestamp_usec = 1;
    optional bool  stream_end            = 2;
}


// Signal of frequency change. Client -> Server.
message Signal {
	optional bool signal = 1;
}
